dbt/include/snowflake/macros/materializations/incremental.sql

```diff
@@ -11,20 +11,39 @@
        for (presumably) faster overall incremental processing.
 
        Low-level specifics:
+       If an invalid option is specified, then we will raise an
+       excpetion with corresponding message.
+
        Languages other than SQL (like Python) will use a temporary table.
        With the default strategy of merge, the user may choose between a temporary
        table and view (defaulting to view).
-       The append strategy can use a view because it will run a single INSERT statement.
-       When the unique_key is none,
-       then we can use a view because it will run a single INSERT statement.
+
```

Comment:
We should add this back in:
```suggestion
       The append strategy can use a view because it will run a single INSERT statement.

```